posthog/temporal/data_imports/pipelines/pipeline/delta_table_helper.py

```diff
@@ -124,12 +125,19 @@ def write_to_deltalake(
                 normalize_column_name(x) for x in primary_keys if normalize_column_name(x) in py_table_column_names
             ]
 
-            delta_table.merge(
-                source=data,
-                source_alias="source",
-                target_alias="target",
-                predicate=" AND ".join([f"source.{c} = target.{c}" for c in normalized_primary_keys]),
-            ).when_matched_update_all().when_not_matched_insert_all().execute()
+            merge_stats = (
+                delta_table.merge(
+                    source=data,
+                    source_alias="source",
+                    target_alias="target",
+                    predicate=" AND ".join([f"source.{c} = target.{c}" for c in normalized_primary_keys]),
+                )
+                .when_matched_update_all()
+                .when_not_matched_insert_all()
+                .execute()
+            )
+
+            self._logger.debug(f"Delta Merge Stats: {json.dumps(merge_stats)}")
```

Comment:
Doesn't make much sense to `json.dumps` here without making the whole message a valid JSON, since it will not be parse-able either way.

Reply:
This will all end up as a string value within a `message` key.

Reply:
Anyways, not a big deal either way I think